plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -341,7 +351,31 @@ public Instance startService(String namespace,
         }
     }
 
-    @Override
+    // sets the home directory for a user defined with CHE_USER
+    private void changeWorkingDirOwner(String containerId, LineConsumer machineLogger) throws MachineException {
+        String prepareCmd = "CHE_USER=`id -un`;"
+                + "sudo chown -R ${CHE_USER}:docker /home/user;"
```

Comment:
What if user is root in a container? His home folder will be /root instead of /home/user. Or even if user has different username - not the "user".